docs/source/api/staticdnsentries.rst

```diff
@@ -54,6 +54,10 @@ Request Structure
 	+-------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
 	|      typeId       | no       | Return only static DNS entries of the type identified by this integral, unique identifier                                                  |
 	+-------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+	| limit             | no       | Choose the number of results to return                                                                                                     |
+	+-------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+	| offset            | no       | The number of results to skip before beginning to return results                                                                           |
```

Comment:
Does this endpoint not support orderby/sortOrder?

Reply:
It does, that was an oversight. I'll fix that